files/bin/mount-bpf-fs

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+SYSTEMD_UNIT_DIR="/etc/systemd/system"
+SYSTEMD_UNIT="sys-fs-bpf.mount"
+SYSTEMD_UNIT_PATH="$SYSTEMD_UNIT_DIR/$SYSTEMD_UNIT"
+MOUNT_POINT="/sys/fs/bpf"
+FS_TYPE="bpf"
+
+MOUNT_BPF_FS_DEBUG=${MOUNT_BPF_FS_DEBUG:-false}
+function debug() {
+  if [ "$MOUNT_BPF_FS_DEBUG" = "true" ]; then
+    echo >&2 "DEBUG:" "$@"
+  fi
+}
+
+if mount | grep "type $FS_TYPE"; then
+  debug "$FS_TYPE filesystem already mounted!"
+  exit 0
+elif mount | grep "$MOUNT_POINT"; then
```

Comment:
Will never get here because of the above check.

Reply:
The first check is looking for `type bpf`, this check is looking for `/sys/fs/bpf`. The debug output in the unit tests looks correct to me; am I missing something?